bin/envconfig

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+# setup lsstsw environment
+#
+# source this file from your ~/.bashrc or ~/.zshrc
+#
+# relative to <lsstsw>/bin/
+
+print_error() {
+  >&2 echo -e "$@"
+}
+
+usage() {
+  if [[ ! -z "$1" ]]; then
+    print_error "$0: $1"
+  fi
+
+  # note that heredocs are prefixed with tab chars
+  echo "$(cat <<-EOF
+
+		Usage: $0  [-i] [-r] [-b] [-h]
+
+		Specific options:
+        -i          interactive, propose a list of available environment
+        -r REF      environment name
+        -b bXXXX    activate the environment already used for the build with id bXXXX
+        -h          show this message
+
+		EOF
+  )"
+
+  finish=true
+   
+}
+
+parse_args() {
+  local OPTIND
+  local opt
+
+  while getopts "hrib" opt; do
+    case "$opt" in
+    b)
+      shift;
+      BUILD_ID=$1
+      ;;
+    i)
+      INTERACTIVE=true
+      ;;
+    h)
+      usage
+      ;;
+    r)
+      shift;
+      ENVREF=$1
+      ;;
+    *)
+      usage "Unknown option: ${opt}"
+      ;;
+    esac
+  done
+  shift $((OPTIND-1))
+}
+
+##############
+
+INTERACTIVE=false
+ENVREF=""
+BUILD_ID=""
+
+parse_args "$@"
+
+if [[ !$finish ]]; then
+
+
+if [[ -z $ZSH_NAME ]]; then
+  LSSTSW=$(cd "$(dirname "${BASH_SOURCE[0]}")/.."; pwd) || return 1
+  SUFFIX='sh'
+else
+  LSSTSW=$(cd "$(dirname "$0")/.."; pwd) || return 1
+  SUFFIX='zsh'
+fi
+
+if [[ ! -f "$LSSTSW/eups/current/bin/setups.$SUFFIX" ]]; then
+  echo "error: eups not found in $LSSTSW/eups/current" 1>&2
+  echo "  you may need to [re]run bin/deploy to [re]deploy EUPS." 1>&2
+  return
+fi
+
+export PATH="$LSSTSW/miniconda/bin:$PATH"
+export PATH="$LSSTSW/lfs/bin:$PATH"
+export PATH="$LSSTSW/bin:$PATH"
+
+export MANPATH="$LSSTSW/lfs/share/man:"
+
+# need to activate the miniconda base environment to get the correct list of environments
+source activate "${LSSTSW}/miniconda"
+
+if $INTERACTIVE ; then
+  # get the list of available environments
+  #ENVS="`conda env list | grep -v ^# | awk -F ' /' '{ print $2 }'`"
+  IFS=$'\n' raw_envs=( $( conda env list | grep -v "^#" | grep -v "^ " | grep -v "^base" ) )
+  envs=()
+  #echo "${ENVS}"
+  l=0
+  for i in "${raw_envs[@]}"; do
+    ((l++))
+    envs[l]=`echo "${i}" | cut -f 1 -d ' '`
+    echo "${l} - ${envs[${l}]}"
+  done
+  read -p "Choose environment [1 to ${l}]: " eid
+  LSST_CONDA_ENV_NAME="${envs[${eid}]}"
+  source activate "$LSST_CONDA_ENV_NAME"
+
+elif [[ $ENVREF != "" ]]; then
+  echo "Setting $ENVREF environment..."  
+  LSST_CONDA_ENV_NAME=${ENVREF}
+  source activate "$LSST_CONDA_ENV_NAME"
+
+elif [[ $BUILD_ID != "" ]]; then
+  echo "Looking for evironment used in build ${BUILD_ID}..."
+  if [[ -f "build/${BUILD_ID}.env" ]]; then
+    LSST_CONDA_ENV_NAME=`cat "build/${BUILD_ID}.env" | grep 'environment_name' | cut -f 2 -d ' '`
+    source activate "$LSST_CONDA_ENV_NAME"
+    echo "Environment ${LSST_CONDA_ENV_NAME} found."
```

Comment:
Be more consistent with messaging ("Activat[ing]" or "Setting" above and below, but just "found" here).